user_groups/groups.jsonp

```diff
@@ -792,5 +792,17 @@ process_groups([
     "description":"This is group for anyone who wanted to learn or already know functional programming. Be it Haskell, Scala, F#, Clojure or even Elm. It is more about learning functional concepts and use it in real life application. That includes but not limited to Research, Machine Learning, Business and Web Applications. It is all about learning and growing community working around functional programming languages. Looking forward to all programming enthusiastic who wanted to learning Functional Concepts and Functional Programming language.",
     "lat":18.5204,
     "lng":73.8567
+},
+{
+    "name":"SpringsFP",
```

Comment:
Can you alphabetize this? (see first item here https://github.com/fsharp/community.fsharp.org/blob/gh-pages/USER_GROUPS.md )